addok_france/utils.py

```diff
@@ -160,10 +160,15 @@ def add(labels, label):
 def match_housenumber(helper, result):
     if not helper.check_housenumber:
         return
+    tokens = []
     for token in sorted(helper.tokens, key=lambda t: t.position):
-        if token.kind == 'housenumber' and token in result.housenumbers:
-            data = result.housenumbers[str(token)]
-            result.housenumber = data.pop('raw')
-            result.type = 'housenumber'
-            result.update(data)
-            break
+        if token.kind == 'housenumber':
```

Comment:
If I understand correctly we have to be sure the housenumber is at first position(s?). I thought the position wasn't relevant?